workflow/run_simulation.rb

```diff
@@ -11,11 +11,21 @@
 basedir = File.expand_path(File.dirname(__FILE__))
 
 def run_workflow(basedir, rundir, hpxml, debug, timeseries_output_freq, timeseries_outputs, skip_validation, add_comp_loads,
-                 output_format, building_id, ep_input_format)
+                 output_format, building_id, ep_input_format, schedules_type)
```

Comment:
Do we want more than just the `schedules_type` argument?

Reply:
Hmm. I will have to think more about what we want to do with the run_simulation.rb script.